releases/v1.7.0-beta.toml

```diff
@@ -0,0 +1,95 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.6.0"
+
+pre_release = true
+
+preface = """\
+The eighth major release of containerd includes new functionality alongside many improvements.
+This release is intended to be the last major release of containerd 1.x before 2.0.
+Some functionality in this release may be considered experimental or unstable, but will become stable or default in 2.0.
+This release still adheres to our backwards compability guarantees and users who do not use or enable new functionality should use this release with the same stability expectations.
+The previous 1.6 release has also become a long term stable release for users who prefer releases with mostly stability improvements and wish to wait a few releases for new functionality.
+
+_This is a beta release and includes some functionality which is not yet complete. While most APIs are finalized before merge, they are subject to change until the official release._
+
+### Highlights
+
+#### Sandbox API
+
+The sandbox API provides a new way of managing containerd's shim, providing more flexibility and functionality for multi-container environments such as Pods and VMs.
+This API makes it easier to manage these groups of containers at a higher level and offers new extension points for shim implementations and clients.
+
+* **Sandbox API** ([#6703](https://github.com/containerd/containerd/pull/6703))
+* **CRI Sandbox API Implementation** ([#7228](https://github.com/containerd/containerd/pull/7228))
+
+#### Transfer Service _(in progress)_
+
+* **Transfer Service** ([#7320](https://github.com/containerd/containerd/pull/7320))
```

Comment:
Seems to need a brief explanation

Reply:
It will, leaving it empty in early beta until merged